### DISCUSSION TITLE
求 消逝的光芒

### DISCUSSION THREAD

**Discussion by lcc-666** [Gamuxorg/bbs#393] [投递和请求游戏]
我有消逝的光芒linux版，求汉化

**Comment by hyh19962008** [comment]
可以先提供一下游戏本体吗？

**Reply by lcc-666** [reply]
本体我已经删了，但是链接还在
https://rutracker.org/forum/dl.php?t=5290892
用的是俄罗斯的网站

**Comment by hyh19962008** [comment]
已上传站内
官方是有中文的，但是你下载的这个版本比较老还不支持

**Reply by lcc-666** [reply]
我之前下载没注意，刚刚去毛子网站找了个比较新的 已经支持中文了
https://rutracker.org/forum/dl.php?t=6000580
